src/controllers/controller.cpp

```diff
@@ -121,7 +121,7 @@ void Controller::receive(const QByteArray data, mixxx::Duration timestamp) {
             continue;
         }
         function.append(".incomingData");
-        QScriptValue incomingData = m_pEngine->resolveFunction(function);
+        QScriptValue incomingData = m_pEngine->wrapFunctionCode(function);
```

Comment:
incomingData functions take 2 arguments -- not the 5 MIDI handlers take

Maybe you could add a "num arguments" to wrapFunctionCode and then create the argument list with a loop (the names of the arguments dont matter so you could make them like 'arg1', 'arg2', 'arg3', etc.)